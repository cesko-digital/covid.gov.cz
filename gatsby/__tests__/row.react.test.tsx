import React from 'react';
import renderer from 'react-test-renderer';

import Row from '@/components/row';

test('Row with content renders correctly', () => {
  const component = renderer.create(<Row>content</Row>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Row with additional class names renders correctly', () => {
  const component = renderer.create(<Row className="className">content</Row>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
