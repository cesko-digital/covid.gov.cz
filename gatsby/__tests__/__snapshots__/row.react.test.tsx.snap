// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row with additional class names renders correctly 1`] = `
<div
  className="row className"
>
  content
</div>
`;

exports[`Row with content renders correctly 1`] = `
<div
  className="row "
>
  content
</div>
`;
